### PR TITLE
Updates of CSIPSTR requriments and text around it

### PR DESCRIPTION
**Pull request by karinbredenberg**
Updates of CSIPSTR following disucssion in meetings on 14 and 15th of November. Decision regarding the METS document and the simple case is needed.

### CODE REVIEW

File: specification/implementation/index.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ title: Implementation of the CSIP
...
 
-As explained above, any implementation is destined to be outdated sooner or later. However, the authors of the CS IP have made their best effort to reuse already available best practices and established core standards, and to carry out intensive discussions within the digital preservation community. All of the above should guarantee that the implementation can be used with only minor updates (for example minor updates to metadata elements) throughout the next few decades.
+As explained earlier, any implementation using a metadata standard is destined to be outdated sooner or later. However, the authors of the CS IP have made their best effort to reuse already available best practices and established core standards, and to carry out intensive discussions within the digital preservation community. All of the above should guarantee that the implementation can be used with only minor updates (for example minor updates to metadata elements) throughout the next few decades.
```
Comment (carlwilson) [inline specification/implementation/index.md]: is destined to be outdated sooner or later
to
will inevitably become obsolete.

File: specification/implementation/structure/folders/index.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Folder structure of the CSIP
+title**: Folder structure of the CSIP
```
Comment (carlwilson) [inline specification/implementation/structure/folders/index.md]: Remove `**`

File: specification/implementation/structure/folders/index.md [hunk 3]
```diff
@@ -12,49 +12,49 @@ The CS IP folder structure is presented in Figure 8 below. The structure follows
...
+**CSIPSTR10**: The `representations` folder **MUST** include a sub-folder for each individual representation (i.e. the “representation folder”) named with a string uniquely identifying the representation within the scope of the package (for example the name of the representation and/or its creation date could be good examples for an representation sub-folder).
 
 <a name="CSIPSTR11"></a>
-CSIPSTR11: The representation folder MUST include a sub-folder named `data` which includes all data constituting the representation.
+**CSIPSTR11**: The representation folder **MUST** include a sub-folder named `data` which includes all data constituting the representation.
 
 <a name="CSIPSTR12"></a>
-CSIPSTR12: The representation folder SHOULD include a metadata file named `METS.xml` which then MUST include information about the identity and structure of the representation and its components.
+**CSIPSTR12**: The representation folder **MUST** include a metadata file named `METS.xml` which then includes information about the identity and structure of the representation and its components.
```
Comment (carlwilson) [inline specification/implementation/structure/folders/index.md]: which then includes
becomes
which includes

File: specification/implementation/structure/implementation/index.md [hunk 4]
```diff
@@ -2,10 +2,12 @@
 title: Implementing the structure
 ---
 ## 4.2.	Implementing the structure
-The requirements presented in Section 4.1 leave room for quite a few decisions during implementation. For the sake of clarity we provide here examples for two extremes – the simplest and the full use of the structure.
+The requirements presented in Section 4.1 leave room for decisions during the implementation. For the sake of clarity we here provide examples for two extremes – the simplest and the full use of the structure.
```
Comment (carlwilson) [inline specification/implementation/structure/implementation/index.md]: we here provide examples
to
we provide examples

File: specification/implementation/structure/index.md [hunk 5]
```diff
@@ -2,13 +2,12 @@
 title: CSIP structure
 ---
 # 4.	CSIP structure
-The preferred implementation of the conceptual model described in Principle 3.6 is a fixed physical (folder) structure which follows exactly the conceptual structure. While the CS IP doesn't prohibited alternative implementations of the conceptual model such implementations aren't recommended.
+The preferred implementation of the conceptual model described in Principle 3.6 http://earkcsip.dilcis.eu/specification/specification/principles/#principle-36 is a fixed physical (folder) structure which follows exactly the conceptual structure. While the CS IP doesn't prohibited alternative implementations of the conceptual model such implementations aren't recommended.
```
Comment (carlwilson) [inline specification/implementation/structure/index.md]: Principle 3.6 http://earkcsip.dilcis.eu/specification/specification/principles/#principle-36
to
[Principle 3.6](http://earkcsip.dilcis.eu/specification/specification/principles/#principle-36)

**Comment by karinbredenberg** [comment]
From Björn:
There MUST exist at least one METS-file per IP regardless if it is typed as an IP, SIP, AIP, DIP, AIU or AIC. However, there exist at least two scenarios:

    if a representation could/must exist as an independent representation it requires a representation METS-file so therefore it is a MUST
    if an IP, for some reason, is expected to be one single entity regardless the number of representations it contain, it might contain separately representation METS-files, or not as long as there exist a root METS-file. Therefore it is a SHOULD

Regardless the different scenarios, any tools must support both scenarios (our tools already do so).The conclusion is that it must be possible to use either SHOULD or MUST and therefore it's a question addressed over time to be stated in directives and policies. My proposal is to use SHOULD since there always must be required to have one METS-file per IP and it opens up for policy's to state whether or not they should use separate representation METS-files.

File: specification/implementation/index.md (hunk 1 above)
Comment (karinbredenberg) [reply specification/implementation/index.md]: Update made.

File: specification/implementation/structure/folders/index.md (hunk 2 above)
Comment (karinbredenberg) [reply specification/implementation/structure/folders/index.md]: Removed

File: specification/implementation/structure/folders/index.md (hunk 3 above)
Comment (karinbredenberg) [reply specification/implementation/structure/folders/index.md]: Fixed.

File: specification/implementation/structure/implementation/index.md (hunk 4 above)
Comment (karinbredenberg) [reply specification/implementation/structure/implementation/index.md]: Updated

File: specification/implementation/structure/index.md (hunk 5 above)
Comment (karinbredenberg) [reply specification/implementation/structure/index.md]: Fixed.

**Pull request merged**